src/docs/spring/spring-configuring.adoc

```diff
@@ -69,20 +69,20 @@ management.metrics.export.atlas.step=PT1M
 ----
 
 For a full list of configuration properties that can influence Atlas publishing, see
-`com.netflix.spectator.atlas.AtlasConfig`.
+[`com.netflix.spectator.atlas.AtlasConfig`](https://github.com/Netflix/spectator/blob/master/spectator-reg-atlas/src/main/java/com/netflix/spectator/atlas/AtlasConfig.java) in Netflix's API documentation.
```

Comment:
This link can be broken in the future.
We can add the [permalink](https://github.com/Netflix/spectator/blob/15cd93ced5c7f6147f546b639f5ffaccffaff1cd/spectator-reg-atlas/src/main/java/com/netflix/spectator/atlas/AtlasConfig.java) instead but that one will be outdated as they update the file.

Reply:
That's a constant risk with documentation that refers to someone else's documentation. All we can do is make sure it's right today and expect an issue in the future. It's one of several reasons why we have to go back over the docs from time to time.